wiki/Linux-Troubleshooting.md

```diff
@@ -65,6 +65,9 @@ Put `DRI_PRIME=1 %command%` into SteamVR's commandline options and in those of a
 
 Put `__NV_PRIME_RENDER_OFFLOAD=1 __VK_LAYER_NV_optimus=NVIDIA_only __GLX_VENDOR_LIBRARY_NAME=nvidia %command%` into SteamVR's commandline options and in those of all VR games you intend to play with ALVR.
 
+### SteamVR Dashboard not rendering in VR
```

Comment:
Please specify that this is for nvidia

Reply:
done, it is possible that this issue could arise with an AMD dGPU too and could be similarly solved with `DRI_PRIME=1 steam-runtime` or whatever the equivalent is on AMD systems but I don't have one to be able to confirm.